### ISSUE TITLE
Maven project never compiles if sometime the .classpath is not here

### ISSUE THREAD

**Issue by andxu** [redhat-developer/vscode-java#658]
![image](https://user-images.githubusercontent.com/17075863/46242393-3187ef80-c3fb-11e8-8ce0-f25d40854f84.png)

**Comment by fbricon** [comment]
Yes, the eclipse settings files are vital pieces of the project. So this is expected. I'm open to suggestions to improve the UX, but they'll still produce these errors when missing, as long as eclipse.jdt.ls is based of off Eclipse.
